presto-docs/src/main/sphinx/connector/snowflake.rst

```diff
@@ -0,0 +1,111 @@
+===============
+Snowflake Connector
+===============
+
+The Snowflake connector allows querying and creating tables in an external
+Snowflake database. This can be used to join data between different
+systems like Snowflake and Hive, or between two different Snowflake accounts.
+
+Configuration
+-------------
+
+To configure the Snowflake connector, create a catalog properties file
+in ``etc/catalog`` named, for example, ``snowflake.properties``, to
+mount the Snowflake connector as the ``snowflake`` catalog.
+Create the file with the following contents, replacing the
+connection properties as appropriate for your setup:
+
+.. code-block:: none
+
+    connector.name=snowflake
+    connection-url=jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?db={dbname}
+    connection-user=root
+    connection-password=secret
+
+The ``connection-url`` defines the connection information and parameters to pass
+to the Snowflake JDBC driver.
+
+.. code-block:: text
+
+    connection-url=jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?db={dbname}
+
+The ``connection-user`` and ``connection-password`` are typically required and
+determine the user credentials for the connection, often a service user.
+
+Multiple Snowflake Databases or Accounts
+^^^^^^^^^^^^^^^^^^^^^^
+
+The Snowflake connector can only access a single database within a Snowflake account.
+Thus, if you have multiple Snowflake databases, or want to connect to multiple
+Snowflake accounts, you must configure multiple instances of the Snowflake connector.
```

Comment:
Just commenting about "you must configure multiple instances of the Snowflake connector": this kind of clarity to the user is great!